Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -1073,6 +1073,15 @@ void WebProcessProxy::createGPUProcessConnection(IPC::Connection::Handle&& conne
     protectedProcessPool()->createGPUProcessConnection(*this, WTFMove(connectionIdentifier), WTFMove(parameters));
 }
 
+void WebProcessProxy::resetGPUProcess()
+{
+    WEBPROCESSPROXY_RELEASE_LOG(Process, "resetGPUProcess: webPage=%p, pageProxyID=%" PRIu64 ", webPageID=%" PRIu64 " hasReset=%d", &webPage, webPage.identifier().toUInt64(), webPage.webPageID().toUInt64(), m_hasResetGPUProcess);
```

Comment:
This should at the very least be a RELEASE_LOG_ERROR, maybe even a RELEASE_LOG_FAULT.